src/vsr/client.zig

```diff
@@ -13,7 +13,7 @@ const Message = @import("../message_pool.zig").MessagePool.Message;
 const IOPS = @import("../iops.zig").IOPS;
 const FIFO = @import("../fifo.zig").FIFO;
 
-const log = std.log.scoped(.client);
+const log = stdx.log.scoped(.client);
```

Comment:
Did you mean to use `log.mark` somewhere?

Reply:
Nope, this was just to prevent tests from failing due when a `log.err` is hit.

```
pub const log = if (builtin.is_test)
    // Downgrade `err` to `warn` for tests.
    // Zig fails any test that does `log.err`, but we want to test those code paths here.
```

Reply:
Ahh, right!